.github/CODEOWNERS

```diff
@@ -46,5 +46,8 @@
 # URI Options
 /source/uri-options/ @mongodb/dbx-spec-owners-uri-options
 
+# Workforce OIDC
+/docs/workforce-human-oidc-auth.md @mongodb/dbx-spec-owners-workforce-oidc
```

Comment:
reminder to create the team

Reply:
It does, and I'm not sure why GH is complaining: https://github.com/orgs/mongodb/teams/dbx-spec-owners-workforce-oidc

Reply:
Ah, looks like the parent team needs to be set to dbx-spec-reviewers, I've requested that + the request is pending rn